tests/integration/functional/cli/test_secret.py

```diff
@@ -35,7 +35,10 @@ def test_create_secret():
     with cleanup_secrets() as secret_name:
         result = runner.invoke(
             secret_create_command,
-            [secret_name, "--test_value=aria", "--test_value2=axl"],
+            [
+                secret_name,
+                '--values={"test_value":"aria","test_value2":"axl"}',
+            ],
```

Comment:
You need to test the new option _in addition_ to the current one. Both options are supported.